### ISSUE TITLE
Tensorflow installation for python version 3.6

### ISSUE THREAD

**Issue by preetivyas** [tensorflow/tensorflow#14890]
I am trying to install tensorflow framework on my Windows 10 system and the framework is showing me the similar error of 
![image](https://user-images.githubusercontent.com/31855851/33243068-c272b944-d2ac-11e7-97fc-327b8b18bafa.png)
I have checked my system architecture which is 32 bit. I have tried it installing using the conda - 3.5 python version fix but then again same error arises. Please help me resolve this issue as I am on a fix for last few days.

**Comment by prabufca** [comment]
Hi Preti,

Kindly uninstall python 3.6 version and install python 3.5 version

**Comment by aselle** [comment]
@preetivyas , does this suggestion resolve your issue?

**Comment by preetivyas** [comment]
So after reading all such blogs I decided to install python version 3.5 and 2.7 too. I was installing the tensorflow using conda installation as given here.
![image](https://user-images.githubusercontent.com/31855851/33348787-ae8620dc-d465-11e7-83fc-f6a954a11344.png)
I installed it successfully but after installation when I went inside my python terminal and tested tensorflow it shows me this.
![image](https://user-images.githubusercontent.com/31855851/33348934-13a13826-d466-11e7-94d6-4a2e72dfd657.png)

But after a while I thought of following this way which gave me an exception in the end.
![image](https://user-images.githubusercontent.com/31855851/33348966-32f707fa-d466-11e7-9b23-d47f84a5278d.png)

And now it runs tensorflow!
